build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -eo pipefail
+
+BUNDLE="bundle"
+MIDDLEMAN="middleman"
+
+if ! which ${BUNDLE} > /dev/null; then \
+  echo -e "${red} ${BUNDLE} not installed, please install Bundler (http://bundler.io) ${NC}"
+  exit 1; \
+fi
+
+if ! which ${MIDDLEMAN} > /dev/null; then \
```

Comment:
You probably don't want to check for a rubygem being present before you have actually installed it using bundler.